mypy.ini

```diff
@@ -0,0 +1,41 @@
+# Global options
+
+[mypy]
+python_version = 3.5
+
+follow_imports = skip
+
+; Strictness settings
+; disallow_any_unimported = True
+; disallow_any_expr = True
+; disallow_any_decorated = True
+; disallow_any_explicit = True
+; disallow_any_generics = True
+; disallow_subclassing_any = True
+
+; disallow_untyped_calls = True
+; disallow_untyped_defs = True
+; disallow_incomplete_defs = True
+; disallow_untyped_decorators = True
+
+; no_implicit_optional = True
+; no_strict_optional = False
+
+; Enable all warnings
+; warn_redundant_casts = True
+; warn_unused_ignores = True
+; warn_return_any = True
+; warn_unreachable = True
+
+; allow_untyped_globals = False
+; strict_equality = True
+
+[mypy-numpy.*]
+ignore_missing_imports = True
+
+[mypy-plumpy.*]
+ignore_missing_imports = True
```

Comment:
Is this similar to `pylint` issuing missing import warnings if it is not being run in the "system" environment? If so, would it make sense to also run `mypy` under system?

Reply:
Mypy.. has its own "import system" - the problem here is that when dependencies don't have type annotations, that will be reported as a failure.

For now I'm not sure this makes a difference since we _also_ use ``skip`` for the ``follow_imports`` option, but we definitely want to get rid of that one eventually.

The import mechanism is described here: https://mypy.readthedocs.io/en/stable/running_mypy.html#how-mypy-handles-imports